app/views/works/show.html.erb

```diff
@@ -1,12 +1,22 @@
 <%= render Elements::HeaderComponent.new(level: :h1, text: @work_form.title, classes: 'mb-4 d-inline-block') %>
 <span class="ms-4 fst-italic status"><%= @status_presenter.status_message %></span>
 <%= render Elements::ButtonLinkComponent.new(link: edit_work_path(@work_form.druid), label: 'Edit or deposit', classes: 'float-end', variant: :'outline-primary') if @status_presenter.editable? %>
+
+<%= render Elements::Tables::TableComponent.new(id: 'files-table', classes: 'table-work mb-5', label: 'Files') do |component| %>
+    <% component.with_header(headers: %w[Filename Description], each_classes: ['col-6']) %>
+    <% @content.content_files.each do |content_file| %>
+        <% component.with_row(values: [content_file.filename, content_file.label]) %>
+    <% end %>
```

Comment:
We could also consider slot collection rendering. Not sure if clearer or less clear:

```suggestion
    <% component.with_rows(
         @content.content_files.map do |content_file|
           { values: [content_file.filename, content_file.label] }
         end
    ) %>
```

Reply:
Hmmm ... I pondered over this. While generally I'd advocate for using VCs collection support, in this case I think it is a tad more confusing because you have to generate an array of hashes as the collection to pass in.

Reply:
Yep, I buy it.